src/Development/IDE/Compat.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE CPP #-}
+module Development.IDE.Compat
+    (
+        getProcessID
+    ) where
+
+#ifdef mingw32_HOST_OS
+
+import qualified System.Win32.Process as P (getCurrentProcessId)
```

Comment:
This import is failing since you did not add a dependency on `Win32`.

Reply:
happy to see windows azure ci in action! :smile_cat: